src/ffi.rs

```diff
@@ -804,15 +857,15 @@ extern "C" {
 extern "C" {
     /// Deallocate any resources associated with an image
     ///
-    /// * `image` —          image to be destroyed
+    /// * 'image' —          image to be destroyed
     pub fn opj_image_destroy(image: *mut opj_image_t);
 }
 extern "C" {
     /// Creates an image without allocating memory for the image (used in the new version of the library).
     ///
-    /// * `numcmpts` —     the number of components
-    /// * `cmptparms` —    the components parameters
```

Comment:
I mean the change from `` ` `` to `'` in these doccomments

Reply:
Hm...I didn't do any format, so I think this change is due to bindgen version difference.